lib/fidor_api/client/dsl/transfers/generic.rb

```diff
@@ -30,7 +30,7 @@ def update_transfer(id, attributes = {}, options = {})
 
           def confirm_transfer(id, options = {})
             check_transfer_support! :generic
-            request(:put, "/transfers/#{id}/confirm", {}, options.delete(:headers)).headers['Location']
+            request(:put, "/transfers/#{id}/confirm", {}, options[:headers])
```

Comment:
I'm wondering if maybe we should keep it a non breaking change by still returning the URL ...
Something in the direction of:

```ruby
request(:put, "/transfers/#{id}/confirm", {}, options[:headers]).body.dig(:links, :redirect)
```

Reply:
But that would cut API clients off from using
* Location or links/status to check the status of the confirmation
* Using other parts of the response body to for example check if the action was already exempted

However we could add an additional method to keep clients compatible, and add a deprecation warning?

Reply:
> However we could add an additional method to keep clients compatible, and add a deprecation warning?

Nah ... then we'll go and have a breaking change with a fresh minor release mentioned in the change-log. I think currently not much people are using this methods at all.